lib/expressMetrics.js

```diff
@@ -0,0 +1,16 @@
+const redis = require('./redis')
+
+const metricsPrefix = 'metrics-'
+
+module.exports = {
+  metricsPrefix,
+  metricsMiddleware() {
+    return function (req, res, next) {
+      const path = req.path
+      if (path !== '/metrics') {
+        redis.incr(`metrics-${req.path}.callCount`)
```

Comment:
Right now I'm just incrementing the call count but we could do whatever here.

Reply:
`express-metrics` counted the requests to the route, and not the exact path, i.e. a request to `/car?number=null` would affect the `/car` key in the metrics, and `/weather/forecasts/en` affect `/weather/forecasts/:lang?`.

We probably want to do something similar. I wouldn't want to expose the exact paths requested.

Also, we should have separate counts for different methods. Especially GET and OPTIONS is pretty important so we don't count double when browsers do a cors request.

Reply:
Yup! `req.path` is not the full path but just the path without the search params, so this is how it currently works:

![image](https://user-images.githubusercontent.com/318208/32049483-79cd5cd4-ba4d-11e7-8a5d-d08b24fe6986.png)

Reply:
Ah, I forgot that express extracts the querystring. Then disregard that part :)

Reply:
>Also, we should have separate counts for different methods. Especially GET and OPTIONS is pretty important so we don't count double when browsers do a cors request.

Seems like the `OPTION` requests don't reach the middleware since, I think, it's caught by the cors middleware before that. In any case I added the request method in 75bb9a9